packages/http-core/src/http-error.js

```diff
@@ -122,6 +134,36 @@ function parseObject(body) {
   return message;
 }
 
+/**
+ * @param {object} body
+ * @private
+ * @returns {string}
+ */
+function parseDescription(body) {
```

Comment:
I think only SparkHttpError should have the rawDescription field, since it's the one that adds the tracking id and url to the message. on that note, `rawMessage` might be a better field name.